cpp/unreal_plugins/SpCore/Source/SpCore/Unreal.cpp

```diff
@@ -254,6 +254,7 @@ std::string Unreal::getPropertyValueAsString(const Unreal::PropertyDesc& propert
         property_desc.property_->IsA(FFloatProperty::StaticClass())  ||
         property_desc.property_->IsA(FDoubleProperty::StaticClass()) ||
         property_desc.property_->IsA(FStrProperty::StaticClass())    ||
+        property_desc.property_->IsA(FNameProperty::StaticClass())    ||
```

Comment:
- whitespace not aligned but otherwise correct
- sensibly placed immediately after string, which is a thoughtful choice 👌